ui/views.py

```diff
@@ -109,6 +108,68 @@ def register():
         return _render_template('join.html', form=form)
 
 
+@app.route('/recognize', methods=['POST', 'GET'])
+def recognize():
+    form = UploadToRecognizeForm()
+
+    if request.method == 'POST':
+
+        if not form.validate_on_submit():
+            image_errors = form.image.errors
+            for error in image_errors:
+                flash(error, 'danger')
+            return _render_template('index.html', form=form)
+
+        file = form.image.data
+        files = {file.name: (file.filename, file)}
+        r = requests.post(api_url + '/neural/recognize',
+                          headers=headers,
+                          files=files)
+
+        # Successful upload
+        if r.status_code == 200:
+            flash('You have been successfully uploaded your image', 'info')
```

Comment:
"Your image has been successfully uploaded"
or
"You have successfully uploaded your image"